src/declarations/stencil-private.ts

```diff
@@ -2569,25 +2569,26 @@ export type TelemetryCallback = (...args: any[]) => void | Promise<void>;
  * The model for the data that's tracked.
  */
 export interface TrackableData {
-  yarn: boolean;
-  component_count?: number;
   arguments: string[];
-  targets: string[];
-  task: TaskCommand | null;
+  build: string;
+  component_count?: number;
+  config: Config;
+  cpu_model: string | undefined;
   duration_ms: number | undefined;
-  packages: string[];
-  packages_no_versions?: string[];
+  has_app_pwa_config: boolean;
+  isBrowserEnv: boolean;
```

Comment:
can we keep this as `snake_case` for consistency with the rest of the properties?

I don't recall off the top of my head _why_ it needs to be like this. I know we don't have any adapter b/w this and the data store (so it gets stored literally as you see it here). That alone isn't much of an issue, unless there's something in the serialization I'm not remembering 🤔

Reply:
(I did check the notes from the original impl, and couldn't find anything. Lars is OOO, otherwise I'd ping him).

Reply:
consistency is reason enough 👍